aws/rust-runtime/aws-config/external-types.toml

```diff
@@ -9,17 +9,16 @@ allowed_external_types = [
    "aws_credential_types::provider::SharedCredentialsProvider",
    "aws_sdk_sts::types::_policy_descriptor_type::PolicyDescriptorType",
    "aws_smithy_async::rt::sleep::AsyncSleep",
+   "aws_smithy_async::rt::sleep::SharedAsyncSleep",
+   "aws_smithy_async::time::SharedTimeSource",
    "aws_smithy_async::time::TimeSource",
-   "aws_smithy_client::bounds::SmithyConnector",
-   "aws_smithy_client::conns::default_connector::default_connector",
-   "aws_smithy_client::erase::DynConnector",
-   "aws_smithy_client::erase::boxclone::BoxCloneService",
-   "aws_smithy_client::http_connector::ConnectorSettings",
-   "aws_smithy_client::http_connector::HttpConnector",
    "aws_smithy_http::body::SdkBody",
    "aws_smithy_http::endpoint",
    "aws_smithy_http::endpoint::error::InvalidEndpointError",
    "aws_smithy_http::result::SdkError",
+   "aws_smithy_runtime_api::client::dns::SharedDnsResolver",
+   "aws_smithy_runtime_api::client::http::SharedHttpClient",
+   "aws_smithy_runtime_api::shared::IntoShared",
```

Comment:
do we want `aws_smithy_runtime_api::*`?

Reply:
I think that in practice we would allow all API crate members, but I like listing them explicitly because it's interesting to see a list of everything in the API crate that a runtime crate touches.